gethhook/geth_test.go

```diff
@@ -34,10 +34,22 @@ func (r *TestChainContext) Engine() consensus.Engine {
 
 func (r *TestChainContext) Config() *params.ChainConfig { return r.chainConfig }
 
+func (r *TestChainContext) CurrentHeader() *types.Header {
+	return &types.Header{}
+}
+
 func (r *TestChainContext) GetHeader(hash common.Hash, num uint64) *types.Header {
 	return &types.Header{}
 }
 
+func (r *TestChainContext) GetHeaderByNumber(num uint64) *types.Header {
+	return &types.Header{}
+}
+
+func (r *TestChainContext) GetHeaderByHash(hash common.Hash) *types.Header {
+	return &types.Header{}
+}
```

Comment:
why are we adding these extra receiver methods? https://github.com/OffchainLabs/nitro/blob/v3.9.0/gethhook/geth_test.go#L27-L39

Reply:
[ad4ffef](https://github.com/Layr-Labs/nitro/pull/141/commits/ad4ffeffeed69fd882fdc04e8a907ea3530f8542)